### DISCUSSION TITLE
Show off DALL·E 3 Creations in LLM's

### DISCUSSION THREAD

**Discussion by H0llyW00dzZ** [ChatGPTNextWeb/NextChat#3378] [Show and tell]
> [!NOTE]  
> This models are currently in preview and their availability to the public is limited.

![img-xNOXRXAnAnJPYKa2Ifpqq3Dp](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/5c6c7f0c-c200-4b22-b40d-f84162c1541e)

![img-CcSutbeBbiBpbU3aAdgRrL5i](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/6ea18ff0-0376-4b5d-88d5-45abad4e0d58)

![img-lKLookd4644kdwxOBMmi7j9r](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/b68b769c-fd60-4fa4-8c7d-4fdd385a49e6)

![img-ofKhCafigCkavJf5dHb3xhOJ](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/2f7caafa-43b0-43af-9540-774a3be8cbb7)

**Comment by H0llyW00dzZ** [comment]
Another

- Manga/Anime Style

![img-Hy77Vb8I2WSaghb5RO41nP4N](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/65c06e03-1e7d-4cda-8cfa-1e9340de27d3)

**Comment by Agilan989** [comment]
This models are currently in preview and their availability to the public is limited.

**Reply by H0llyW00dzZ** [reply]
yes have to contact support if using service provider openai I think, 
because it not showing here

![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/614c84b9-c988-440c-a630-fb46f7eac90d)

**Comment by H0llyW00dzZ** [comment]
Another
- Manga/Anime Style


![img-dtLPDwmakHobi9EGg9QdJFNY](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/0b3a15ee-430d-42b9-8243-2987d05b78da)
![img-b3nZG4GpKJBeaUJkwEpunGGu](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/8f844a5f-3eda-4265-997e-a804cd305e77)
![img-S6mXebl3FD7nVPcFJemk5z7F](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/b66e269d-c498-4532-bafe-2715689b6f7f)

> [!NOTE]  
> All of my DALL·E 3 Creations are used prompts not using seeds and it's automated generated it in LLM's Methods

**Comment by H0llyW00dzZ** [comment]
More I'm so cool

![img-OpfrOVZkvDfjPEGQt2k4FID0](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/e0e35632-57a3-487e-994c-8ea2b6582d7e)
![img-RXUVra2NY298vprL5uVdjeXi](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/61bd4d5e-6a11-44b7-b235-63956bdb38f9)
![img-1UsA2GRugpFSgykinMZ01Rpn](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/9bf03688-7933-4df9-9692-1a92ce8f01cf)

**Comment by H0llyW00dzZ** [comment]
- Female Golang characters depicted in anime style manage complex tasks on Kubernetes

![img-dcoSXxSzq7p7uO10e9KKMJ39](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/72ab1549-13e3-45a7-a3e9-e8fef2beced5)

![golang](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/5166888c-fb92-4df3-a31b-9490f7a80136)

**Reply by ArciAndres** [reply]
Which version of the repo is that one? I cannot find the option to generate images. And I have tried with the GPT-4 API options. 

![image](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/23021023/12db6f54-8490-4fb8-bdfb-8923f7583726)

These are the options I have, but no images are generated. 
![image](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/23021023/a2670a9a-975c-4cca-af5d-951c3e536851)

**Reply by H0llyW00dzZ** [reply]
It's on my forks 

https://hackerchat.btz.sh/

**Reply by mynumu** [reply]
hey have you got a github for this??

**Reply by H0llyW00dzZ** [reply]
https://github.com/H0llyW00dzZ/ChatGPT-Next-Web

**Comment by H0llyW00dzZ** [comment]
Another 
- Female Golang characters depicted in anime style manage complex tasks on Kubernetes

![img-gDNWvy2ACmPSIg7FFQSDLwOC](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/f8e67167-1314-454b-ba38-a09355b40783)

**Comment by H0llyW00dzZ** [comment]
I am so cool

![img-SRmSOFHq1tawpGh6fH9RjK3v](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/17626300/d2456559-ac80-488e-91b1-059523cd9d89)
